genai-perf/tests/test_console_exporter.py

```diff
@@ -288,49 +296,236 @@ def test_invalid_goodput_output(self, monkeypatch, capsys) -> None:
+            "│ └───────────┴────────┴─────┴─────┴─────┴─────┴─────┘          │\n"
+            "│                 Energy Consumption (MJ)                       │\n"
+            "│ ┏━━━━━━━━━━━┳━━━━━━┳━━━━━━┳━━━━━━┳━━━━━━┳━━━━━━┳━━━━━━┓       │\n"
+            "│ ┃ GPU Index ┃  avg ┃  min ┃  max ┃  p99 ┃  p90 ┃  p75 ┃       │\n"
+            "│ ┡━━━━━━━━━━━╇━━━━━━╇━━━━━━╇━━━━━━╇━━━━━━╇━━━━━━╇━━━━━━┩       │\n"
+            "│ │      gpu0 │ 0.55 │ 0.50 │ 0.60 │ 0.60 │ 0.59 │ 0.57 │       │\n"
+            "│ └───────────┴──────┴──────┴──────┴──────┴──────┴──────┘       │\n"
+            "└───────────────────────────────────────────────────────────────┘\n"
+            "            NVIDIA GenAI-Perf | Memory Metrics             \n"
+            "┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓\n"
+            "┃                                                         ┃\n"
+            "┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩\n"
+            "│                  GPU Memory Used (GB)                   │\n"
+            "│ ┏━━━━━━━━━━━┳━━━━━━┳━━━━━━┳━━━━━━┳━━━━━━┳━━━━━━┳━━━━━━┓ │\n"
+            "│ ┃ GPU Index ┃  avg ┃  min ┃  max ┃  p99 ┃  p90 ┃  p75 ┃ │\n"
+            "│ ┡━━━━━━━━━━━╇━━━━━━╇━━━━━━╇━━━━━━╇━━━━━━╇━━━━━━╇━━━━━━┩ │\n"
+            "│ │      gpu0 │ 4.00 │ 4.00 │ 4.00 │ 4.00 │ 4.00 │ 4.00 │ │\n"
+            "│ └───────────┴──────┴──────┴──────┴──────┴──────┴──────┘ │\n"
+            "│               Total GPU Memory (GB)                     │\n"
+            "│ ┏━━━━━━━━━━━┳━━━━━━┳━━━━━┳━━━━━┳━━━━━┳━━━━━┳━━━━━┓      │\n"
+            "│ ┃ GPU Index ┃  avg ┃ min ┃ max ┃ p99 ┃ p90 ┃ p75 ┃      │\n"
+            "│ ┡━━━━━━━━━━━╇━━━━━━╇━━━━━╇━━━━━╇━━━━━╇━━━━━╇━━━━━┩      │\n"
+            "│ │      gpu0 │ 8.00 │ N/A │ N/A │ N/A │ N/A │ N/A │      │\n"
+            "│ └───────────┴──────┴─────┴─────┴─────┴─────┴─────┘      │\n"
+            "└─────────────────────────────────────────────────────────┘\n"
+            "       NVIDIA GenAI-Perf | Utilization Metrics       \n"
+            "┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓\n"
+            "┃                                                   ┃\n"
+            "┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩\n"
+            "│                GPU Utilization (%)                │\n"
+            "│ ┏━━━━━━━━━━━┳━━━━━┳━━━━━┳━━━━━┳━━━━━┳━━━━━┳━━━━━┓ │\n"
+            "│ ┃ GPU Index ┃ avg ┃ min ┃ max ┃ p99 ┃ p90 ┃ p75 ┃ │\n"
+            "│ ┡━━━━━━━━━━━╇━━━━━╇━━━━━╇━━━━━╇━━━━━╇━━━━━╇━━━━━┩ │\n"
+            "│ │      gpu0 │  78 │  75 │  80 │  80 │  80 │  79 │ │\n"
+            "│ └───────────┴─────┴─────┴─────┴─────┴─────┴─────┘ │\n"
+            "└───────────────────────────────────────────────────┘\n"
+        )
+
+        returned_data = capsys.readouterr().out
+        assert returned_data == expected_content
+
+    def test_invalid_telemetry_verbose(self, monkeypatch, capsys) -> None:
```

Comment:
what is the test trying to check? the test name sounds like we should be catching some error but I only see normal console output checks.

Reply:
I removed tests that do not contain telemetry data as they are covered by other tests as @dyastremsky suggested.